packages/shared/src/stateChainGateway/index.ts

```diff
@@ -79,4 +86,17 @@ export const getRedemptionDelay = (
   return stateChainGateway.REDEMPTION_DELAY();
 };
 
+export const getPendingRedemption = async (
+  accountId: `0x${string}`,
+  networkOpts: FundingNetworkOptions,
+): Promise<PendingRedemption | undefined> => {
+  const stateChainGateway = getStateChainGateway(networkOpts);
+  const pendingRedemption =
+    await stateChainGateway.getPendingRedemption(accountId);
+
+  return pendingRedemption.amount > 0
+    ? stateChainGateway.getPendingRedemption(accountId)
+    : undefined;
+};
```

Comment:
not clear why we're doing it like this?

Reply:
do you mean the `pendingRedemption.amount > 0`? thats because there are no `null` values in solidity. if something is not set, it has the initial value for the type: https://www.wtf.academy/en/solidity-start/InitialValue/